### PR TITLE
Update signin.client.view.html

### PR DESCRIPTION
**Pull request by copyhold**
Login: make signup button at login page more prominent #891 
as a solution to [this](https://github.com/Trustroots/trustroots/pull/891)

### CODE REVIEW

File: modules/users/client/views/authentication/signin.client.view.html [hunk 1]
```diff
@@ -49,42 +49,40 @@
...
+                   tabindex="4">
+                  Forgot?
+                  <span ng-if="auth.authError" class="hidden-xs">
+                    Recover your password
+                  </span>
+                </a>
               </div>
             </fieldset>
           </form>
+          <div class="text-white">
```
Comment (petr-hajek) [inline modules/users/client/views/authentication/signin.client.view.html]: it is missing hidden-xs class so it is displayed twice in the mobile view
selection_007

File: modules/users/client/views/authentication/signin.client.view.html [hunk 2]
```diff
@@ -49,42 +49,40 @@
...
-                <span ng-if="auth.authError" class="hidden-xs">
-                  Recover your password
-                </span>
-              </a>
-              <div class="text-center form-group">
-                <br><br>
+              <div class="form-group">
                 <button type="submit"
-                        class="btn btn-default btn-action"
+                        class="btn btn-default btn-lg"
```
Comment (petr-hajek) [inline modules/users/client/views/authentication/signin.client.view.html]: I'd leave it as it was so Login button is bigger

File: modules/users/client/views/authentication/signin.client.view.html (hunk 1 above)
Comment (petr-hajek) [reply modules/users/client/views/authentication/signin.client.view.html]: Also the comment `<!-- For bigger screens -->` should be before this div

File: modules/users/client/views/authentication/signin.client.view.html (hunk 2 above)
Comment (copyhold) [reply modules/users/client/views/authentication/signin.client.view.html]: `btn-action` looks too big to my taste. It is also not a Bootstrap standard.

Comment (simison) [reply modules/users/client/views/authentication/signin.client.view.html]: >It is also not a Bootstrap standard.

Bootstrap is just a base framework and not a design system per-se, it's okay to have additional components. ;-)

`btn-action` is big indeed, it's for "call to action" type of interactions and not designed to be used in forms.

File: modules/users/client/views/authentication/signin.client.view.html [hunk 3]
```diff
@@ -72,7 +72,8 @@
               </div>
             </fieldset>
           </form>
-          <div class="text-white">
+          <!-- For bigger screens -->
+          <div class="text-white hidden-hs">
```
Comment (petr-hajek) [inline modules/users/client/views/authentication/signin.client.view.html]: It is class hidden-xs
Now it is still displaying both

File: modules/users/client/views/authentication/signin.client.view.html (hunk 2 above)
Comment (petr-hajek) [reply modules/users/client/views/authentication/signin.client.view.html]: OK, fair enough

File: modules/users/client/views/authentication/signin.client.view.html (hunk 3 above)
Comment (copyhold) [reply modules/users/client/views/authentication/signin.client.view.html]: 7a1b22f6ae92ffc04df004b818f2341ce4ce5c2c

**Pull request merged**